test/WebJobs.Script.Tests.Perf/Artifacts/PS/build-jar.ps1

```diff
@@ -0,0 +1,11 @@
+﻿$tempFolderName = [System.IO.Path]::GetFileNameWithoutExtension([System.IO.Path]::GetTempFileName())
+$tempFolder = [System.IO.Path]::Combine([System.IO.Path]::GetTempPath() ,$tempFolderName)
+[System.IO.Directory]::CreateDirectory($tempFolder)
+
+Write-Host "tempFolder:$tempFolder"
+
+Copy-Item -Path ".\..\Java\*" -Destination $tempFolder -Recurse
+
+& mvn clean package --file "$tempFolder"
```

Comment:
Use the package command from azure-functions-java-worker repo otherwise console output is too verbose